README.md

```diff
@@ -6,6 +6,8 @@ Compared to the original ChorusRF Lamptimer this ESP32 version only requires one
 
 Hardware construction is also simplified as both parts are 3.3v logic and there is no need for level shifting or resistors.  
 
+[Click HERE for a video showing the full setup process to get Chorus32 running on your ESP32 using Arduino IDE.](https://www.youtube.com/watch?v=ip2HUVk_lMs). 
+
```

Comment:
```suggestion
<p align="center">
    <a href="https://www.youtube.com/watch?v=ip2HUVk_lMs"><img src="https://i.ytimg.com/vi/ip2HUVk_lMs/hqdefault.jpg"></a>
</p>
```

Reply:
Very awesome!

Reply:
I like that more, to be honest. I held off on doing so for my initial pull request, because I'm new here and didn't want to clutter things too much, but if you think the video is helpful, then visibility would be good.